docs/pages/tools/flamegraphs.md

```diff
@@ -130,25 +134,25 @@ sed -r -e "/( __libc_start| uv_| LazyCompile | v8::internal::| node::| Builtins_
 
 ### List of Web References
 
-- <https://www.brendangregg.com/flamegraphs.html>
-- <https://nodejs.org/en/docs/guides/diagnostics-flamegraph>
-- <https://netflixtechblog.com/netflix-flamescope-a57ca19d47bb>
-- <https://jaanhio.me/blog/nodejs-flamegraph-analysis/> (this was a great one about filtering methodology)
-- <https://medium.com/voodoo-engineering/node-js-and-cpu-profiling-on-production-in-real-time-without-downtime-d6e62af173e2>
+- https://www.brendangregg.com/flamegraphs.html
+- https://nodejs.org/en/docs/guides/diagnostics-flamegraph
+- https://netflixtechblog.com/netflix-flamescope-a57ca19d47bb
+- [https://jaanhio.me/blog/nodejs-flamegraph-analysis/](this was a great one about filtering methodology)
```

Comment:
Not sure this is necessary.  The [] and () are showing up in the rendered content.  probably can just remove them.